project/Dependencies.scala

```diff
@@ -5,8 +5,11 @@ object Dependencies {
   val `scala-compact-collection` = "org.scala-lang.modules" %% "scala-collection-compat" % "2.8.1"
 
   val zioVersion     = "2.0.2"
+  val zioPreludeVersion = "1.0.0-RC16"
+
   val zio            = "dev.zio" %% "zio"          % zioVersion
   val `zio-streams`  = "dev.zio" %% "zio-streams"  % zioVersion
+  val `zio-prelude`  = "dev.zio" %% "zio-prelude"  % zioPreludeVersion
```

Comment:
Should be moved to the ftp dependencies. While it's RC I would like to keep it out of core dependencies.